poethepoet/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.11.0b3"
+__version__ = "0.12.0b0"
```

Comment:
```suggestion
__version__ = "0.11.0b4"
```
We're still building up to 0.11.0 :(